build.sbt

```diff
@@ -4228,6 +4228,8 @@ lazy val `engine-runner` = project
               "net.snowflake.client",
               "com.sun.jna",
               "com.tableau.hyperapi",
+              "com.typesafe.config.impl.ConfigImpl$EnvVariablesHolder",
+              "com.typesafe.config.impl.ConfigImpl$SystemPropertiesHolder",
```

Comment:
I manually checked that this also fixes overriding the `LANGUAGE_SERVER_YDOC_HOSTNAME` when running the docker container